client/components/RepositoryList.jsx

```diff
@@ -28,14 +28,30 @@ export default class RepositoryList extends Component {
     this.props.filterRepos(evt.target.value)
   }
 
+  filteredRepositories() {
```

Comment:
Well, since props are immutable and read-only I think it's ok. Plus the method is just used internally and adding an argument makes the code a bit longer. What do you think?